Python-path-traversal-files-main-final/MatteoGuadrini/Butterfly-Backup/bb.py

```
@@ -0,0 +1,2831 @@
+    return necessaries_retries
+
+
+def start_process(command):
+    """Start rsync commands
+
+    :param command: rsync list command
+    :return: command
+    """
+    fd = get_std_out()
+    if fd == "STDOUT":
+        p = subprocess.call(command, shell=True)
```

Ruff (0.13.3)

`subprocess` call with `shell=True` identified, security issue

(S602)